### PR TITLE
Term entry swiftui tracking

### LINKED ISSUE #1512: [Term Entry] SwiftUI View Modifier: `.tracking()`
### Reason/inspiration (optional)

Please write a new entry under: content/swiftui/concepts/viewmodifier/terms/ about the modifier `.tracking()` in Swift.

A "Syntax" section with pseudo code snippet.
An "Example" section with a swift code snippet and image to accompany. Images should go in the media/ folder.

To get started with the PR and entry file:

Under content/swiftui/concepts/viewmodifier/terms/, create a new folder called `tracking`. Then create a new file called `tracking.md` under content/swiftui/concepts/viewmodifier/terms/`tracking`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new entry for SwiftUI ViewModifier: .tracking()
- Closes #1512 

### Type of Change

- Adding a new entry

### Checklist

- [ x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

###Additional Information
I am using npm; I have node 18.12.
I was unable to run format:verify (prettier is not recognized)
I was unable to run lint:md (markdownlint is not recognized)
I was unable to run test (jest is not recognized)

### CODE REVIEW

File: content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.tracking()`** modifier method modifies the spacing of the characters in a [`Text`](https://www.codecademy.com/resources/docs/swiftui/views/text) view.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md:16]:
```suggestion
The **`.tracking()`** modifier method <alters or adjusts> the spacing of the characters in a [`Text`](https://www.codecademy.com/resources/docs/swiftui/views/text) view.
```

Can we change there verb here to one of the two options I provided?

File: content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Syntax
+
+```pseudo
+Text("Some text")
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md:21]: Yes, I actually much prefer this way of showing the syntax for the ViewModifiers. I'm going to create an issue to amend these for the other existing entries that don't already following this structure if you would be down to take that on. But no pressure, if not, I can go ahead and make that edit myself but do lmk!

File: content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    .tracking(x)
+```
+
+The `.tracking()` modifier method takes one parameter, `x`, entered as an integer to define the space to be added to or deducted from each character's default spacing in `pt` (points). SwiftUI applies the system's default value if `x` is given `0` as a value or no parameter is given. A positive value for `x` results in additional spacing, whereas a negative value results in decreased spacing.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md:25]: Can we break this down into an unordered list?

File: content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+> **Note:** `.tracking()` is not the same as `.kerning()`. Although both the `.tracking()` and `.kerning()` modifier methods modify the spacing of the characters, they behave differently. `.tracking()` will pull apart the ligatures and does not leave trailing whitespace, while `.kerning()` will maintain ligatures and leaves some trailing whitespace.
+
+## Example
+
+In the example below, the `.tracking()` modifier method is called on three `Text` views with decreased, default and increased tracking values.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md:31]:
```suggestion
In the example below, the `.tracking()` modifier method is called on three `Text` views with decreased, default, and increased tracking values.
```

Oxford comma (Docs style preference)

**Review by yangc95 — COMMENTED** [review]
@cslylla Thanks for creating this so quickly! Conducted the first review

File: content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md (hunk 2 above)
Comment (cslylla) [reply content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md:21]: Sure, I can do that!

File: content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md (hunk 1 above)
Comment (cslylla) [reply content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md]: I changed it to adjust.

File: content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md (hunk 3 above)
Comment (cslylla) [reply content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md]: I updated it to a bullet list. 👍🏻

**Review by yangc95 — APPROVED** [review]
Looks good for a second review!

**Review by SSwiniarski — APPROVED** [review]
@cslylla, @yangc95 Looks good to merge as is.

**Comment by SSwiniarski** [comment]
@cslylla congrats on the new entry! You can find the live entry here: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/tracking

image

**Pull request merged**